CRT.c

```diff
@@ -734,3 +743,88 @@ void CRT_setColors(int colorScheme) {
 
    CRT_colors = CRT_colorSchemes[colorScheme];
 }
+
+void CRT_handleSIGSEGV(int signal) {
+   CRT_done();
+
+   fprintf(stderr, "\n\n"
+      "FATAL PROGRAM ERROR DETECTED\n"
+      "============================\n"
+      "Please check at https://htop.dev/issues whether this issue has already been reported.\n"
+      "If no similar issue has been reported before, please create a new issue with the following information:\n"
+      "\n"
+      "- Your htop version (htop --version)\n"
+      "- Your OS and kernel version (uname -a)\n"
+      "- Your distribution and release (lsb_release -a)\n"
+      "- Likely steps to reproduce (How did it happened?)\n"
+#ifdef HAVE_EXECINFO_H
+      "- Backtrace of the issue (see below)\n"
+#endif
+      "\n"
+   );
+
+   fprintf(stderr,
+      "Error information:\n"
+      "------------------\n"
+      "A signal %d was received.\n"
```

Comment:
Maybe also translate with `strsignal(3)`, in case signal have weird numbers on some system?